base/reflection.jl

```diff
@@ -1198,14 +1222,18 @@ end
 
 # high-level, more convenient method lookup functions
 
-# type for reflecting and pretty-printing a subset of methods
+"""
+`MethodList` is a type for reflecting and pretty-printing a subset of methods. It is returned by
+[`methods`](@ref).
+"""
 mutable struct MethodList <: AbstractArray{Method,1}
     ms::Array{Method,1}
     mt::Core.MethodTable
 end
 
 size(m::MethodList) = size(m.ms)
 getindex(m::MethodList, i::Integer) = m.ms[i]
+collect(m::MethodList) = m.ms
```

Comment:
This new method eliminates the copy that `collect` typically makes. We already have `collect(::MethodList)::Vector{Method}`
```suggestion
```